openfl/display/BitmapData.hx

```diff
@@ -453,7 +453,7 @@ class BitmapData implements IBitmapDrawable implements IBitmapData {
 				src_display_object.__update(false, true);
 			}
 		}
-		__drawGL ( renderSession, source, matrix, clipRect, smoothing, !__usingPingPongTexture, false, false);
+		__drawGL ( renderSession, source, matrix, clipRect, smoothing, false/*!__usingPingPongTexture*/, false, false);
```

Comment:
commented code

Reply:
Yep, submitted like this so that we don't forget to discuss this case ;) I want to be sure it is ok to always use `false` here before this PR gets merged.

This is a "breaking" change in the sense it could modify some behavior but I don't think we ever used it in a way that would "break". Let me explain it.

The only special case is when not using pingpong textures. In this case, `GLBitmap.drawBitmapDrawable()` will render the current bitmap data (`target`) to the currently bound render target (a fresh one created in `BitmapData.pushFrameBuffer()`). This is a discrepancy vs when using pingpong textures for which we do not perform anything similar.
I should also be noted that we can call `BitmapData.draw()` only once without using pingpong textures (after the first popFrameBuffer, the flag is set to true).
So this could only alter the visuals of a non empty bitmap data that was filled without going through `draw()` (i.e. by filling its image buffer manually such as for color lookups) and for which we would like to overwrite part of it with the rendering of a display object.

I don't think this ever happens.

Please let me know if you think about something I have overlooked.

Reply:
Well I trust you on this one ;)

Reply:
;)